lib/peep.rb

```diff
@@ -0,0 +1,34 @@
+require 'pg'
+
+class Peep
+  attr_reader :id, :title, :peep
+
+  def initialize(id:, title:, peep:)
+    @id = id
+    @title = title
+    @peep = peep
+  end
+
+  def self.all
+
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else
+      connection = PG.connect(dbname: 'chitter')
+    end
+
```

Comment:
This is a good start for the next steps.  you could look at mapping the results to new instances of the Peep class rather than just returning an array of peeps